codecs/av1_packet.go

```diff
@@ -1,7 +1,7 @@
 package codecs
 
 import (
-	"github.com/pion/rtp/v2/pkg/obu"
+	"github.com/pion/rtp/pkg/obu"
```

Comment:
In my understanding, `master` branch is v2 and `v1` branch is v1.
Is it intended to use v1/pkg/obu from v2/codecs?